src/System.Security.Cryptography.X509Certificates/tests/Configurations.props

```diff
@@ -2,12 +2,14 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
FWIW I'm working on cleaning up the .props, test .csproj because most of our newer tests are not running due build changes and renaming of netstandard :(